app/controllers/home_controller.rb

```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
     def index
+        @posts = Post.order(created_at: :desc).page(params[:page]).per(10)
```

Comment:
You could use a constant for page number. It would make it clearer and easier to change pagination.